python/lsst/cp/pipe/defects.py

```diff
@@ -716,3 +762,154 @@ def run(self, inputDefects, camera):
+
+
+class MergeDefectsCombinedTask(MergeDefectsTask):
+    """Task to measure defects in combined images."""
+
+    ConfigClass = MergeDefectsCombinedTaskConfig
+    _DefaultName = "cpDefectMergeCombined"
+
+    def runQuantum(self, butlerQC, inputRefs, outputRefs):
+        inputs = butlerQC.get(inputRefs)
+        # Turn inputDefects and secondaryInputDefects into a list
+        # which is what MergeDefectsTask expects.
+        tempList = [inputs['inputDefects'], inputs['secondaryInputDefects']]
+        # Rename inputDefects
+        inputsCombined = {'inputDefects': tempList, 'camera': inputs['camera']}
+
+        outputs = super().run(**inputsCombined)
+        butlerQC.put(outputs, outputRefs)
+
+
+class MeasureDefectsCombinedConnections(MeasureDefectsConnections,
+                                        dimensions=("instrument", "detector")):
+    inputExp = cT.Input(
+        name="defectExps",
+        doc="Input ISR-processed combined exposures to measure.",
+        storageClass="Exposure",
+        dimensions=("instrument", "detector"),
+        multiple=False,
+        isCalibration=True,
+    )
+    camera = cT.PrerequisiteInput(
+        name='camera',
+        doc="Camera associated with this exposure.",
+        storageClass="Camera",
+        dimensions=("instrument", ),
+        isCalibration=True,
+        lookupFunction=lookupStaticCalibration,
+    )
+
+    outputDefects = cT.Output(
+        name="combinedExpDefects",
+        doc="Output measured defects.",
+        storageClass="Defects",
+        dimensions=("instrument", "detector"),
+    )
+
+
+class MeasureDefectsCombinedTaskConfig(MeasureDefectsTaskConfig,
+                                       pipelineConnections=MeasureDefectsCombinedConnections):
+    """Configuration for measuring defects from combined exposures.
+    """
+    pass
+
+
+class MeasureDefectsCombinedTask(MeasureDefectsTask):
+    """Task to measure defects in combined images."""
+
+    ConfigClass = MeasureDefectsCombinedTaskConfig
+    _DefaultName = "cpDefectMeasureCombined"
+
+
+class MeasureDefectsCombinedWithFilterConnections(MeasureDefectsCombinedConnections,
+                                                  dimensions=("instrument", "detector")):
+    """Task to measure defects in combined flats under a certain filter."""
+    inputExp = cT.Input(
+        name="defectExps",
+        doc="Input ISR-processed combined exposures to measure.",
+        storageClass="Exposure",
+        dimensions=("instrument", "detector", "physical_filter"),
+        multiple=False,
+        isCalibration=True,
+    )
+    camera = cT.PrerequisiteInput(
+        name='camera',
+        doc="Camera associated with this exposure.",
+        storageClass="Camera",
+        dimensions=("instrument", ),
+        isCalibration=True,
+        lookupFunction=lookupStaticCalibration,
+    )
+
+    outputDefects = cT.Output(
+        name="combinedExpDefects",
```

Comment:
This name should be the default instead of in the pipeline (and should match `cpPartialDefectsFromFlatCombined ` or whatever we settle on).